target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -100,6 +100,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan:2" "2:lan:1" "3:lan:3" "4:lan" "0:wan" "6@eth0"
 		;;
+	dlink,dwr-960)
+		ucidef_add_switch "switch0" \
+			"1:lan" "2:lan" "3:lan" "5:lan" "0:wan" "6@eth0"
```

Comment:
Your LED setting says port 4 is LAN, but this says port 5 is LAN. Which one is correct?